packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -223,12 +231,15 @@ export function TableNG(props: TableNGProps) {
       return rows;
     `;
 
-    const convert = new Function('frame', fnBody);
-
-    const records = convert(frame);
-    return records;
+    // Creates a function that converts a DataFrame into an array of TableRows
+    // Uses new Function() for performance as it's faster than creating rows using loops
+    const convert = new Function('frame', fnBody) as unknown as FrameToRowsConverter;
```

Comment:
```suggestion
    const convert: FrameToRowsConverter = new Function('frame', fnBody) as FrameToRowsConverter;
```

Does this work to avoid the unknown type assertion?

Reply:
Doesn't look like it solves the issue 😞

Reply:
The betterer entry? I just figured this looked a bit cleaner if nothing else. I'll leave it up to you if you think it's worth changing. 😆

Reply:
Ha, let's leave as is for now. It actually doesn't pass the linter that way. I'll need to look into it more.

Reply:
I think there is a way to resolve this with function factory, but I might just be making that up. Sure, we can follow up on this later.